src/extension/app/plugins/delete/delete.js

```diff
@@ -77,6 +78,7 @@ export function createDeletePlugin(appStore) {
               actionCallback,
               appStore.i18n('reload'),
             );
+            appStore.fireEvent(EXTERNAL_EVENTS.RESOURCE_DELETED);
```

Comment:
Changed all plugins to call `fireEvent` directly from the plugin vs in the appStore. This was done to simplify testing.